ui/app/components/mfa/mfa-login-enforcement-form.js

```diff
@@ -41,14 +40,14 @@ export default class MfaLoginEnforcementForm extends Component {
   searchSelectOptions = null;
 
   @tracked name;
-  @tracked targets = A([]);
+  @tracked targets = [];
   @tracked selectedTargetType = 'accessor';
   @tracked selectedTargetValue = null;
   @tracked searchSelect = {
     options: [],
     selected: [],
   };
-  @tracked authMethods = A([]);
+  @tracked authMethods = [];
```

Comment:
Just an aside - I'm not sure this needs to be tracked since it's only set once in the constructor?

Reply:
targets and auth methods are pushed to (with `addObject`, but that is also deprecated and will be removed) so they do need to be tracked

Reply:
Oh - I didn't seen anywhere `addObject` was used for `authMethods` - it looks like it's just passed as options for the `<Select>`